tests/unit/test_login_manager.py

```diff
@@ -228,3 +241,44 @@ def dummy_command(*, login_manager, collection_id):
         return True
 
     assert dummy_command(collection_id=gcs_id)
+
+
+def test_compute_timer_scope_no_data_access():
+    transfer_scope = globus_sdk.scopes.TransferScopes.all
+
+    computed = str(compute_timer_scope())
+    assert computed.startswith(BASE_TIMER_SCOPE)
+    assert computed == f"{BASE_TIMER_SCOPE}[{TRANSFER_AP_SCOPE}[{transfer_scope}]]"
+
+
+def test_compute_timer_scope_one_data_access():
+    transfer_scope = globus_sdk.scopes.TransferScopes.all
+    foo_scope = urlfmt_scope("foo", "data_access")
+
+    computed = str(compute_timer_scope(data_access_collection_ids=["foo"]))
+    assert computed.startswith(BASE_TIMER_SCOPE)
+    assert foo_scope in computed
+    assert (
+        computed
+        == f"{BASE_TIMER_SCOPE}[{TRANSFER_AP_SCOPE}[{transfer_scope}[*{foo_scope}]]]"
+    )
+
+
+def test_compute_timer_scope_multiple_data_access():
+    transfer_scope = globus_sdk.scopes.TransferScopes.all
+    foo_scope = urlfmt_scope("foo", "data_access")
+    bar_scope = urlfmt_scope("bar", "data_access")
+    baz_scope = urlfmt_scope("baz", "data_access")
+
+    computed = str(
+        compute_timer_scope(data_access_collection_ids=["foo", "bar", "baz"])
+    )
+    assert computed.startswith(BASE_TIMER_SCOPE)
+    assert foo_scope in computed
+    assert bar_scope in computed
+    assert baz_scope in computed
+    start_part = f"{BASE_TIMER_SCOPE}[{TRANSFER_AP_SCOPE}[{transfer_scope}["
+    end_part = "]]]"
+    assert computed.startswith(start_part)
+    assert computed.endswith(end_part)
+    assert computed == f"{start_part}*{foo_scope} *{bar_scope} *{baz_scope}{end_part}"
```

Comment:
It seems like only the last line is needed.

Reply:
_scratches head_

Yeah, you'd have to ask Past Stephen why it wasn't just that one. And he's not around anymore.